### DISCUSSION TITLE
Accents for spanish language

### DISCUSSION THREAD

**Discussion by mateomv92** [tecoholic/ner-annotator#110] [Q&A]
Good afternoon! 

I'm fascinated by how easy to work with your program, but I'm having a little problem uploading UTF-8 txt files in Spanish with accents. Is there a proper way to create these txt files or upload them to avoid the question marc symbol instead of the character and the accent? 

Thank you very much!

**Comment by alvi-khan** [comment]
Hello @mateomv92! Could you please provide a bit more detail or some examples for which you're facing issues? I tried with a few sample Spanish sentences and it seemed fine, but I don't know Spanish so maybe I'm misunderstanding your issue.

**Reply by alvi-khan** [reply]
@mateomv92 bumping this.
